grafana-plugin/src/models/user/user.ts

```diff
@@ -82,11 +82,6 @@ export class UserStore extends BaseStore {
     };
   }
 
-  @action
-  getCurrentUser() {
-    return this.items[this.currentUserPk as User['pk']];
-  }
```

Comment:
we already have a `UserStore.currentUser` `getter`.. this didn't seem to be referenced anywhere in the codebase (nor in `oncall-private`), I think it's safe to remove